EIPS/eip-7445.md

```diff
@@ -0,0 +1,421 @@
+---
+eip: 7445
```

Comment:
```suggestion
eip: 7504
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename